builders/account-management/identity.md

```diff
@@ -45,27 +45,34 @@ To store your information on-chain, you must bond some funds, which will eventua
 
 ## Checking Prerequisites { : #checking-prerequisites }
 
-For this guide, you will need the following:
+To follow along with this guide, you will need the following:
 
-- [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://{{ networks.dancelight.dns_name }}#/accounts){target=\_blank} open and connected to the Tanssi Dancelight TestNet
-- At least one account funded with `{{ networks.dancelight.token_symbol }}` tokens
+=== "Tanssi MainNet"
+
+    - The [developer portal](https://polkadot.js.org/apps/?rpc=wss://{{ networks.mainnet.dns_name }}#/accounts){target=\_blank} open and connected to Tanssi MainNet
+    - At least one account funded with `{{ networks.mainnet.token_symbol }}` tokens
+
+=== "Dancelight TestNet"
+    
+    - The [developer portal](https://polkadot.js.org/apps/?rpc=wss://{{ networks.dancelight.dns_name }}#/accounts){target=\_blank} open and connected to the Tanssi Dancelight TestNet
+    - At least one account funded with `{{ networks.dancelight.token_symbol }}` tokens
 
-If you need help importing your accounts into Polkadot.js Apps, please check out the [Connecting to Polkadot.js](/builders/toolkit/substrate-api/wallets/talisman/#connecting-to-polkadotjs){target=\_blank} guide.
+If you need help importing your accounts into the developer portal, please check out the [Connecting to the developer portal](/builders/toolkit/substrate-api/wallets/talisman/#connecting-to-polkadotjs){target=\_blank} guide.
 
 ## Get Started {: #get-started }
 
-Depending on the information to be included, there are a couple of different ways to set and clear an identity using Polkadot.js Apps. If you intend to register your identity using only the default fields, you can follow the instructions for [Managing an Identity via the Accounts UI](#manage-via-accounts). **This is the recommended way to set and manage your identity**.
+Depending on the information to be included, there are a couple of different ways to set and clear an identity using the developer portal. If you intend to register your identity using only the default fields, you can follow the instructions for [Managing an Identity via the Accounts Page](#manage-via-accounts). **This is the recommended way to set and manage your identity**.
```

Comment:
```suggestion
Depending on the information to be included, there are a couple of different ways to set and clear an identity using the developer portal. If you intend to register your identity using only the default fields, you can follow the instructions for [Managing an Identity via the Accounts](#manage-via-accounts) page. **This is the recommended way to set and manage your identity**.
```

The title of the section we're linking to is "Manage an Identity via Accounts" so that can be capitalized, page is not part of the title so it should not be capitalized and does not need to be in the link text (although it could be - no strong preference there)